### DISCUSSION TITLE
[Base] Daylight in Overture

### DISCUSSION THREAD

**Discussion by asuschromeos** [OvertureMaps/data#255] [Discussion]
### Category Feedback

Hello. Daylight has announced that they will now be part of Overture. Their latest release v1.58 is from the OSM snapshopt on Sun 10/20/2024. At what frequency will "Base" part be updated next? For example, what is the OSM snapshopt date for the next Overture release?

**Comment by jwass** [comment]
Good question, @asuschromeos. Despite Daylight no longer being published we are still doing the same quality assurance process and cleanup work behind the scenes. That same data is just now part of Overture. We'll be publishing on the same cadence and the OSM data for each Overture release won't be more than a few weeks old.

**Comment by jwass** [comment]
Closing this but feel free to reopen with any followup questions or start a new one.
